addons/payment_adyen/models/payment_provider.py

```diff
@@ -37,6 +38,12 @@ class PaymentProvider(models.Model):
         help="The base URL for the API endpoints",
         required_if_provider='adyen',
     )
+    adyen_event_url = fields.Char(
+        string="Event URL",
+        help="This URL needs to be pasted on Adyen's website, in Event URLs configuration",
+        readonly=True,
+        default=lambda self: f"{self.get_base_url()}/pos_adyen/notification",
```

Comment:
won't this hardcode the value at the initialisation ? I think it would be better to make it as a compute as the URL might change with time

Reply:
I forgot to set the field as `store=False`.